docs/reference/how-to/disk-usage.asciidoc

```diff
@@ -136,20 +136,40 @@ PUT index
 // CONSOLE
 
 [float]
+=== Watch your shard size
+
+Larger shards are going to be more efficient at storing data. While it is not possible to give a precise recommendation for how large shards should be, and very large shards come with drawbacks (for example, longer recovery times), it is generally possible to have shard sizes in the 20-30 GB range.
```

Comment:
@jasontedor Makes sense. Do you think we should mention an upper range, e.g. 50 GB?